Makefile

```diff
@@ -1016,8 +1016,8 @@ edit:
 pack: hash
 	@echo "$$($(TIME)) Preparing the build pack..."
 	@+{ TMP_DIR=$$(mktemp -d "/tmp/atpack-XXXXXXX"); \
-	    tar -cpvz -X "$(HELPERS_ROOT)/pack-exclude.lst" -f "$${TMP_DIR}/at-$(AT_TODAY).tgz" ./ > \
-	        "$${TMP_DIR}/at-$(AT_TODAY).included" 2>&1; \
+	    tar -cpvz -X "$(HELPERS_ROOT)/pack-exclude.lst" -f "$${TMP_DIR}/at-$(AT_TODAY).tgz" ./ &> \
+	        "$${TMP_DIR}/at-$(AT_TODAY).included"; \
```

Comment:
I'm not sure about this one. I think the intent here is (or should be) to let stderr float until being picked up at line 1024. Maybe instead of `&>`, it should just be `>` and remove the `2>&1`?